src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml

```diff
@@ -0,0 +1,27 @@
+title: CIC Bundle
+filename: cic_case_all_bundle.pdf
+coverpageTemplate: ST_CIC_CoverPage.docx
+hasTableOfContents: true
+hasCoversheets: true
+hasFolderCoversheets: true
+folders:
+  - name: Application Documents
+    documents:
+      - type: documentSet
+        property: /caseDocuments
+        filters:
+          - property: /documentType
+              value: A
+  - name: Further additions
+    documents:
+      - type: documentSet
+        property: /caseDocuments
+        filters:
+          - property: /documentType
+              value: TD
```

Comment:
you can not add multiple "value" like this you need to use "-" and as I remember this property does not support array.

Reply:
example : https://github.com/hmcts/em-ccd-orchestrator/blob/master/src/main/resources/bundleconfiguration/sscs-bundle-config.yaml#L12

https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1201996412#AutomatedBundlingConfigurationFileGuide-FilteringDocuments